CitizenLabDotCo/citizenlab
# CL-700 Content builder e2e test

## Scope
After discussing with Koen D, the scope of this test is:
- Test _each_ column component (rules)
- Test image component by checking that drag and drop works, that it's in the frame, and that it can be deleted
- Every component from the toolbox is tested/placed directly in the frame
- Test drag & drop _within_ the frame by moving a single text component from one container to another
- Check that everything displays correctly on the live view
- Delete all, and then check that everything is deleted in the live page

## Checklist
- [X] Prepared branch for code review

## How urgent is a code review?
End of today or Monday if possible?
